packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -413,18 +445,6 @@ ensureIndex(Posts,
     name: "posts.afSticky_score",
   }
 );
-ensureIndex(Posts,
-  augmentForDefaultView({ metaSticky:-1, score:-1 }),
-  {
-    name: "posts.metaSticky_score",
```

Comment:
Pretty sure this has not been used since we deprecated the "meta" field and meta page.